.github/workflows/ci.yml

```diff
@@ -1,14 +1,14 @@
-name: Check codebase
+name: CI
```

Comment:
I renamed this pipeline (including the file) to get shorter output in GitHub UI. The new setup is aligned [with the HASH repo](https://github.com/hashintel/hash/actions/workflows/ci.yml).